client/go/util/operationResult.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// A struct containing the result of an operation
+// Author: bratseth
+
+package util
```

Comment:
Filenames should use snake case: operationResult.go -> operation_result.go.

Reply:
I couldn't find any consensus on this but ok.